keyboards/ergodash/rev1/keymaps/nopunin10did/keymap.c

```diff
@@ -0,0 +1,6 @@
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+	[0] = LAYOUT_3key_2us(KC_ESC, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_EQL, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_BSPC, KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_PGUP, KC_LBRC, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_ENT, LCA_T(KC_CAPS), KC_A, KC_S, KC_D, KC_F, KC_G, KC_PGDN, KC_RBRC, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT, KC_LSFT, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_RSFT, KC_LCTL, KC_LGUI, KC_LALT, MO(1), KC_GRV, KC_SPC, LT(1,KC_DEL), KC_HOME, KC_END, KC_BSLS, KC_LEFT, KC_RGHT, KC_DOWN, KC_UP),
```

Comment:
Reformats the layers to resemble the layout of the hardware. Makes this easier to read for people.

```suggestion
    [0] = LAYOUT_3key_2us(
        KC_ESC,         KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,                                    KC_EQL,  KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_BSPC, 
        KC_TAB,         KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_PGUP,                                 KC_LBRC, KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_ENT,  
        LCA_T(KC_CAPS), KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_PGDN,                                 KC_RBRC, KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, 
        KC_LSFT,        KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,                                                      KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, 
        KC_LCTL,        KC_LGUI, KC_LALT, MO(1),            KC_GRV,  KC_SPC,  LT(1,KC_DEL),          KC_HOME, KC_END,  KC_BSLS,          KC_LEFT, KC_RGHT, KC_DOWN, KC_UP    
    ),
```